app.js

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const ReactMarkdown = require("react-markdown");
+
+const capitalise = string => {
+  if (string === "aws") return string.toUpperCase();
```

Comment:
In terms of future-proofing, this could be an array of names?

Reply:
Good shout, will do